node/src/components/consensus/era_supervisor/tests.rs

```diff
@@ -47,6 +56,7 @@ enum Event {
     Storage(StorageRequest),
     #[from]
     ContractRuntime(ContractRuntimeRequest),
+    // ConsensusRequest(ConsensusRequest),
```

Comment:
@afck Was this intentional?

Reply:
I guess this can be removed entirely, actually.